recipe/meta.yaml

```diff
@@ -103,6 +103,9 @@ test:
     # Check for bdist_conda
     - python test_bdist_conda_setup.py bdist_conda --help
 
+  downstreams:
+    - conda-smithy
```

Comment:
just curious, how does this work exactly?

Reply:
It will setup an environment and install the package that just got build + the package you wish to test and run the "test" bit of that package's recipe. Here, it would be nice to ensure that new versions of conda-build don't break conda-smithy

https://docs.conda.io/projects/conda-build/en/latest/source/define-metadata.html?highlight=downstream#downstream-tests

Reply:
Ahh cool!

Reply:
It would be good if we could get conda smithy tests running in the recipe